test/productRoutes.test.js

```diff
@@ -0,0 +1,42 @@
+const expect = require("chai").expect;
+const request = require("supertest");
+
+const app = require("../server/main.js");
+const agent = request.agent(app);
+
+const db = require("../server/database/connection.js");
+const Product = require("../server/database/models/product.js");
+
+describe("Products Route:", () => {
+  // before(() => {
+  //   return db.sync({ force: true });
+  // });
+  describe("GET /api/products", () => {
+    it("responds with JSON containing all products", done => {
+      request(app)
+        .get("/api/products")
+        .set("Accept", "application/json")
+        .expect("Content-Type", /json/)
+        .expect(200, done());
```

Comment:
This isn't actually confirming that it received an array of JSON with all products. This is just testing that the response is a JSON.